### ISSUE TITLE
Uninstall

### ISSUE THREAD

**Issue by notDavid** [divamgupta/diffusionbee-stable-diffusion-ui#8]
Hi there,

fyi, [AppCleaner](https://freemacsoft.net/appcleaner/) doesn't find/delete the folder `~/.diffusionbee` (and also there don't seem to be any uninstall instructions.)

So that hidden folder might end up eating a lot of unnecessary space on people's harddrives after removing the app :-)

Would perhaps be better to use the default folder for saving Application data, i assume that's '~/Library/Application Support/DiffusionBee'

**Comment by zorrobyte** [comment]
AppCleaer does a good job and this issue is pretty universal with macOS:
https://freemacsoft.net/appcleaner/


Edit (two years later): I'm an idiot and you literally mentioned AppCleaner* in your initial issue 😆

**Comment by gingerbeardman** [comment]
Just to explain the many thumbs-down: AppCleaner does not pick up the caches at `~/.diffusionbee` 

<img width="493" alt="Screen shot 2022-10-26 at 21 49 18" src="https://user-images.githubusercontent.com/49612/198134703-024f01d1-4951-40cb-96c0-0cbf26302c69.png">

**Comment by gingerbeardman** [comment]
I think the problem is exacerbated by this not being a native app: the code that references the `~/.diffusionbee` is in JavaScript not any native OBjC/Swift places. These uninstaller will not pick it up as it stands.

**Comment by lethargosapatheia** [comment]
> Just to explain the many 👎 ...
> 
> AppCleaner does not do a good job here as it does not spot the caches at `~/.diffusionbee`
> 
> <img alt="Screen shot 2022-10-26 at 21 49 18" width="493" src="https://user-images.githubusercontent.com/49612/198134703-024f01d1-4951-40cb-96c0-0cbf26302c69.png">

I think the most important argument for the thumbsdown is the fact that someone would recommend an app (who knows what privacy issues it might have and so on, but let's ignore that for now) for something as easy as uninstalling a particular app, so only for this case. The process is supposed to be transparent.

If you need a general cleaning-up app, then that might be the way to go, sure.

**Comment by Teloshav** [comment]
Wouldn't it be simpler for the app to have a built in uninstaller? 🤔

**Comment by lethargosapatheia** [comment]
Yes, obviously :) I've no idea if things have changed in the meantime, I haven't used it since my last post.

**Comment by wraybowling** [comment]
they haven't changed. To everyone trying to explain away that some 3rd party tool can or can't detect huge files that the average user will absolutely miss, you're missing the point. The problem is two-fold. First, there's no settings to install the application's bulky models in a custom location like an external hard drive for example. And then, yeah, somewhere also in settings an uninstall button would be kind to the average user.
